wandb/__init__.pyi

```diff
@@ -405,7 +412,10 @@ def finish(exit_code: Optional[int] = None, quiet: Optional[bool] = None) -> Non
 
     Arguments:
         exit_code: Set to something other than 0 to mark a run as failed
-    quiet: Set to true to minimize log output
+        quiet: Set to true to minimize log output
+
+    Returns:
+    None
```

Comment:
don't need

Reply:
it was a hack to "fix" the padding on the last line of the docstring, but I fixed the underlying issue instead.

Reply:
I kno, I removed it